### PR TITLE
Finished the homework 6

### PR DESCRIPTION
**Pull request by MaximIurceac**
Please check my HW

### CODE REVIEW

File: src/main/java/org/example/homework_nr_6/Date.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        this.year = year;
+    }
+
+    public int getDay() {
+        return day;
+    }
+
+    public void setDay(int day) {
+        if( day > 0 && day <=31){
+        this.day = day;
```
Comment (gh3rtzz) [inline src/main/java/org/example/homework_nr_6/Date.java:21]: What will happen if it's out of range?

File: src/main/java/org/example/homework_nr_6/Date.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        }
+    }
+
+    public int getMonth() {
+        return month;
+    }
+
+    public void setMonth(int month) {
+        if( month> 0 && month <= 12){
```
Comment (gh3rtzz) [inline src/main/java/org/example/homework_nr_6/Date.java:34]: What will happen if it's out of range?

File: src/main/java/org/example/homework_nr_6/Date.java (hunk 1 above)
Comment (MaximIurceac) [reply src/main/java/org/example/homework_nr_6/Date.java:21]: Did i forget to put else?

**Review by gh3rtzz — APPROVED** [review]
approved

**Pull request merged**